pages/how-we-work.html

```diff
@@ -5,6 +5,7 @@
 banner_file_mobile: banner--how-we-work-sm.svg
 permalink: /how-we-work
 social_image: social-card--how-we-work.png
+social_image_alt_text: "Navy blue graphic block with gray, blue and yellow squares illustrating sticky notes"
```

Comment:
Suggested text edit:

"Graphic illustration of gray, blue and yellow colored sticky notes arranged in a grid. Some of the notes are blank, some have writing on them, and one has a dog-eared corner."